datacite-draft-doi-handler/src/test/resources/doi_publication_event_valid.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "0",
+  "id": "c2c057c1-67fe-1208-fc21-0401596631ba",
+  "detail-type": "Lambda Function Invocation Result - Success",
+  "source": "lambda",
+  "account": "884807050265",
+  "time": "2020-10-09T09:34:33Z",
+  "region": "eu-west-1",
+  "resources": [
+    "arn:aws:events:eu-west-1:884807050265:event-bus/orestis-event-bus",
+    "arn:aws:lambda:eu-west-1:884807050265:function:orestis-stack-nva-datacite-mds-EventProducer-8IAE7WYCNTPX:$LATEST"
+  ],
+  "detail": {
+    "version": "1.0",
+    "timestamp": "2020-10-09T09:34:33.727Z",
+    "requestContext": {
+      "requestId": "e3f0079c-da29-4e50-92ef-0b699d415e95",
+      "functionArn": "arn:aws:lambda:eu-west-1:884807050265:function:orestis-stack-nva-datacite-mds-EventProducer-8IAE7WYCNTPX:$LATEST",
+      "condition": "Success",
+      "approximateInvokeCount": 1
+    },
+    "requestPayload": {
+      "myRequest": "ListenToMe",
+      "type": "ArbitraryPayload"
+    },
+    "responseContext": {
+      "statusCode": 200,
+      "executedVersion": "$LATEST"
+    },
+    "responsePayload": {
+      "type": "doi.publication",
+      "item": {
+        "id": "https://example.net/unittest/namespace/publication/654321",
+        "doi": "https://doi.org/10.1103/physrevd.100.085005",
+        "doiRequest": {
+          "status": "REQUESTED",
```

Comment:
We discussed yesterday about maybe we need a new `doiRequest.status` for simply DOIs that was created by the user itself that was not part of the curator process.  Because `doiRequest.status == REQUESTED` means for me that this is a request for curator to turn a draft DOI into a findable DOI.

Reply:
Out of scope for this PR. We will look into this in next sprint.